### ISSUE TITLE
leaving zombie processes / no reaping

### ISSUE THREAD

**Issue by kdkaergaard** [berngp/docker-zabbix#24]
On a fairly generic setup, with some standard item.key's, and some External Checks, the container is slowly creating zombie processes.
This seems to be related to the init problem, described in "A correct init process" section, on the [baseimage-docker website](http://phusion.github.io/baseimage-docker/), and in the [pid-1-zombie-reaping-problem](https://blog.phusion.nl/2015/01/20/docker-and-the-pid-1-zombie-reaping-problem/) article.

Just for kicks, i've added a snippet of the Zabbix server, monitoring itself, with 3 monitored hosts.
![zombieprocesses](https://cloud.githubusercontent.com/assets/5849148/10269488/4d328110-6ad9-11e5-8f4f-bb142518c784.PNG)

**Comment by berngp** [comment]
@kdkaergaard true, I have a (branch)[https://github.com/berngp/docker-zabbix/tree/feature/22-supervisord] that actually is meant to capture the effort to move to supervisord and avoid this. Adding syslog, as the link you mentions describes, is also a good idea.

My current problem is just that I have not been able to invest time in this project. Hopefully soon.
